src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -160,7 +164,7 @@ private <CTX> void fetchQueueNamesAndSize(GetQueueStatsRequest<CTX> req, BiConsu
         });
     }
 
-    private <CTX> void fetchRetryDetails(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
+    <CTX> void fetchRetryDetails(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.

Reply:
yes, ok.